src/react-queries/hooks/useIsTradingActive.ts

```diff
@@ -11,43 +11,44 @@ import { thorchainBlockTimeMs } from 'lib/utils/thorchain/constants'
 
 export const useIsTradingActive = ({
   assetId,
-  enabled,
+  enabled = true,
   swapperName,
 }: {
   assetId: AssetId | undefined
-  enabled: boolean
+  enabled?: boolean
   swapperName: SwapperName
 }) => {
-  const [
-    {
-      data: inboundAddressesData,
-      isLoading: isInboundAddressesDataLoading,
-      refetch: refetchInboundAddresses,
-    },
-    { data: mimir, isLoading: isMimirLoading, refetch: refetchMimir },
-  ] = useQueries({
-    queries: [
-      {
-        ...reactQueries.thornode.inboundAddresses(),
-        // Go stale instantly
-        staleTime: 0,
-        // Never store queries in cache since we always want fresh data
-        gcTime: 0,
-        ...(enabled
-          ? {
-              refetchInterval: 60_000,
-              refetchOnWindowFocus: true,
-              refetchOnMount: true,
-            }
-          : {}),
-        select: (data: Result<InboundAddressResponse[], SwapErrorRight>) =>
-          selectInboundAddressData(data, assetId),
-      },
-      {
-        ...reactQueries.thornode.mimir(),
-        staleTime: thorchainBlockTimeMs,
-      },
-    ],
+  const {
+    data: inboundAddressesData,
+    isLoading: isInboundAddressesDataLoading,
+    refetch: refetchInboundAddresses,
+  } = useQuery({
```

Comment:
Review me with whitespace diff hidden - this pretty much splits this into individual queries, with `assetId` type-safety both in `inboundAddresses` and `mimir`